Extension/src/telemetry.ts

```diff
@@ -139,6 +139,15 @@ export function logLanguageServerEvent(eventName: string, properties?: { [key: s
     sendTelemetry();
 }
 
+export async function showStatusBarIntellisenseIndicator(): Promise<boolean> {
```

Comment:
This should be IntelliSense (capital S). I'll submit a PR to fix the pre-existing issues.